cld3.cjs

```diff
@@ -4,10 +4,10 @@ const path = require('path');
 let file = '';
 switch (os.platform()) {
     case 'darwin':
-        file = 'cld3-macos.node';
+        file = 'cld3-darwin-x64.node';
```

Comment:
We will need `arm64` as this is the dev machine for the team but I am not clear on wether GH has it.